docker-compose.yml

```diff
@@ -9,4 +9,9 @@ services:
       context: .
       dockerfile: Dockerfile
     ports:
-      - "8983:8983"
\ No newline at end of file
+      - "8983:8983"
+    volumes:
+      - solr-data:/opt/solr/server/solr/mycores
```

Comment:
I don't think this is the right path for data.
The path is https://github.com/metabrainz/mbsssss/blob/master/common/dataDir.xml#L1

Reply:
For eg. area data path looks like `/opt/solr/server/solr/data/area`
So this needs to be `/opt/solr/server/solr/data`

Reply:
Thanks! Fixed.